### ISSUE TITLE
Create Gingerbread-style grayish icon for the notification bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#148]
```
I have a Google Nexus One with Gingerbread installed and with Gingerbread, 
Google changed the style to a dark notification bar with grayish icons. (see 
attached picture)

It would be great if I could switch the icon of gtalksms to an icon of this 
style to make it blend in with the other icons there.
```

Original issue reported on code.google.com by `mbi...@gmail.com` on 19 Apr 2011 at 10:59
- Merged into: #151

Attachments:
- [gingerbread-notification-bar.jpg](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-148/comment-0/gingerbread-notification-bar.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Hello, 

You can propose some icons. We will be happy to integrate them !
```

Original comment by `Florent....@gmail.com` on 19 Apr 2011 at 12:11

**Comment by GoogleCodeExporter** [comment]
```
Could you please describe on what occasions the yellow and red icons are shown? 
(green = connected/normal operation, I guess?)

If others (more creative) want to try - there are the guidelines: 
http://developer.android.com/guide/practices/ui_guidelines/icon_design_status_ba
r.html#icon9
```

Original comment by `mbi...@gmail.com` on 19 Apr 2011 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
red = disconnected or service not running
orange = connecting, disconnecting, waiting for network to restart the 
connection, waiting to reconnect
green = connected
```

Original comment by `fschm...@gmail.com` on 20 Apr 2011 at 7:39

**Comment by GoogleCodeExporter** [comment]
```
Icon proposals are welcome. :-)
```

Original comment by `fschm...@gmail.com` on 20 Apr 2011 at 1:49
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I've change the icons in version 2.2.
I'll will setup an icon picker for the next version and add icons proposed on 
this thread (if there is any) :)
```

Original comment by `Florent....@gmail.com` on 24 Apr 2011 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
Icon picker is done.

I'm closing this bug and i will open a new one for icons proposal

```

Original comment by `Florent....@gmail.com` on 25 Apr 2011 at 12:39
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.2.1
```

Original comment by `Florent....@gmail.com` on 25 Apr 2011 at 12:43
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 25 Apr 2011 at 12:56
- Changed state: **Moved**
